anthos-bm-gcp-terraform/docs/one_click_install.md

```diff
@@ -0,0 +1,133 @@
+triggers the installation of Anthos on bare metal on the provisioned VMs. The installation
+is triggered from inside the _admin workstation_ VM.
+
+Upon completion the Terraform script will print the following output to the console.
+```sh
+################################################################################
+#          SSH into the admin host and check the installation progress         #
+################################################################################
+
+> gcloud compute ssh tfadmin@cluster1-abm-ws0-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+> tail -f ~/install_abm.log
+
+################################################################################
+```
+
+Use these commands to SSH into the _admin workstation_ and to monitor the installation status.
+Once it is complete the output in the log file will look as follows:
+
+```sh
+...
+[2022-03-22 23:21:12+0000] Moving admin cluster resources to the created admin cluster
+[2022-03-22 23:21:18+0000] Waiting for node update jobs to finish OK
+[2022-03-22 23:22:48+0000] Flushing logs... OK
+[2022-03-22 23:22:48+0000] Deleting bootstrap cluster... OK
+
+Anthos on bare metal installation complete!
+Run [export KUBECONFIG=/home/tfadmin/bmctl-workspace/cluster1/cluster1-kubeconfig] to set the kubeconfig
+Run the [/home/tfadmin/login.sh] script to generate a token that you can use to login to the cluster from the Google Cloud Console
+```
+---
+### Interacting with the Baremetal cluster
+
+You can find your cluster's `kubeconfig` file on the admin machine in the `bmctl-workspace` directory. To verify your deployment, complete the following steps from inside the _admin workstation_.
+
+1. Set the `KUBECONFIG` environment variable with the path to the cluster's configuration file to run `kubectl` commands on the cluster.
+```sh
+export CLUSTER_ID=cluster1
+export KUBECONFIG=$HOME/bmctl-workspace/$CLUSTER_ID/$CLUSTER_ID-kubeconfig
+kubectl get nodes
+```
+
+You should see the nodes of the cluster printed, _similar_ to the output below:
+```sh
+NAME          STATUS   ROLES    AGE   VERSION
+cluster1-abm-cp1-001   Ready    master   17m   v1.18.6-gke.6600
+cluster1-abm-cp2-001   Ready    master   16m   v1.18.6-gke.6600
+cluster1-abm-cp3-001   Ready    master   16m   v1.18.6-gke.6600
+cluster1-abm-w1-001    Ready    <none>   14m   v1.18.6-gke.6600
+cluster1-abm-w2-001    Ready    <none>   14m   v1.18.6-gke.6600
+```
+
+#### Interacting with the cluster via the GCP console
+
+During the setup process, your cluster will be auto-registered in Google Cloud using [Connect](https://cloud.google.com/anthos/multicluster-management/connect/overview). In order to interact with the cluster from the GCP console you must first ***login*** to the cluster.
+
+The [Logging into the Anthos bare metal cluster](login.md) explains how you can do it.
+
+---
+### Cleanup
+
+- Follow the [same cleanup steps as the quickstart guide](quickstart.md#cleanup).
```

Comment:
Thanks for sharing this. Sure I can hear out @askmeegs thoughts as well. 

My thinking was that, this is similar to the variations we have with our other repos like BoA and OB. For example [the extras](https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/main/extras) in the BoA repo has instructions for different tracks. 

I doubt that this will land in devsite. Because it is a copy of an existing quick-starter. So mostly this helps for folks who come to this repo and would like to skip the 2-step process.